### ISSUE TITLE
Ohone gets hanged

### ISSUE THREAD

**Issue by hustleehu** [SubhamTyagi/loco-answers#420]
When i turn on this app M

**Comment by jack3120** [comment]
After I updated my phone trivia hack is not working when I click on get answers the phone hangs if you are reading this pls help me with this problem

**Comment by SubhamTyagi** [comment]
@jack3120 Could provide your device details( RAM, Android version, ROM, Device model etc)?

**Comment by jack3120** [comment]
Device-Redmi y2,3gb ram, Android version 9

**Comment by SubhamTyagi** [comment]
> Device-Redmi y2,3gb ram, Android version 9

Device is looking good: Which OCR are you using? Google vision or Tesseract !

**Comment by jack3120** [comment]
I use Google vision trivia hack was working perfectly in my device before the system update but after the update it is not opening at all

**Comment by SubhamTyagi** [comment]
> before the system update but after the update

Android 9 update?

**Comment by jack3120** [comment]
Yup I think security patch level has increased in this update and thats why it is not opening
